docs/ota-client-guide/modules/ROOT/pages/cross-deploy-images.adoc

```diff
@@ -14,18 +14,18 @@ endif::[]
 :icons: font
 :sectnums:
 
-For our recommended production workflow, you will need to move disk images from one account to another from time to time. For example, you might want to send a development build that you're happy with to the QA team, or send that build to the deployment team once it's passed QA. You can do this with our `garage-deploy` tool.
+For our recommended production workflow, you will need to move disk images from one environment to another from time to time. For example, you might want to send a development build that you're happy with to the QA team, or send that build to the deployment team once it's passed QA. You can do this with our `garage-deploy` tool.
 
 Before you start, make sure that you've installed the xref:install-garage-sign-deploy.adoc[`garage-deploy`] tool first.
 
-To transfer disk images to a different account, follow these steps: ::
+To transfer disk images to a different environment, follow these steps: ::
 . xref:getstarted::generating-provisioning-credentials.adoc[Download provisioning keys] for both accounts.
 +
 We'll assume that you named them `source-credentials.zip` and `dest-credentials.zip`.
 +
```

Comment:
Assuming it works, I would emphasize that the source-credentials.zip is has been generated from Environment A, and dest-credentials.zip has been generated from Envirionment B

Reply:
@xaviergully I am also not certain if it works so I asked @houcros and @patrickvacek to review it. @patrickvacek could you please verify if this might work? Thank you.

Reply:
The current wording seems sufficiently clear for me, where the credentials come from is in their name.

I've just tried it now and it works.

Reply:
@lbonn thank you very much for confirming the changes.